editor/src/components/editor/shortcuts.spec.browser2.tsx

```diff
@@ -1195,7 +1195,7 @@ describe('group selection', () => {
         EP.fromString(`${BakedInStoryboardUID}/${TestSceneUID}/${TestAppUID}:container/bbb`),
       ])
 
-      await expectSingleUndoNSaves(renderResult, 4, async () =>
+      await expectSingleUndoNSaves(renderResult, 2, async () =>
```

Comment:
nitpick, but would you mind replacing these with `expectSingleUndo2Saves`? It would make it easier to mass find how many places are expecting two saves

Reply:
Fixed in d90533219.